integrations/class-woocommerce.php

```diff
@@ -74,12 +74,18 @@ public function affiliate_select_or_input( $checkout ) {
 			return;
 		}
 
-		// get affiliate list
-		$affiliate_list = $this->get_affiliates();
-
 		$description  = affwp_cr_checkout_text();
 		$display      = affwp_cr_affiliate_display();
 		$required     = affwp_cr_require_affiliate();
+		$sorting			= affwp_cr_affiliates_sorting_order();
+
+		// get affiliate list
+		$args = array();
+		if( 'alphabetical' == $sorting ) {
```

Comment:
```suggestion
		if ( 'alphabetical' === $sorting ) {
```
When in doubt, strict comparison.